### PR TITLE
add script changes to accept ui images

### PR DESCRIPTION
**Pull request by bharathappali**
This PR adds the changes to support the kruize ui image being passed via script to launch ui pod along with kruize backend service

### CODE REVIEW

**Review by dinogun — COMMENTED** [review]
Please fix the following
1. Tabs vs Spaces
2. Default image should be the one that matches the KRUIZE_UI_VERSION and not test

File: remote_monitoring_demo/README.md [hunk 1]
```diff
@@ -34,3 +34,8 @@ $ ./remote_monitoring_demo.sh -r -o [kruize image]
 # To terminate the demo
 $ ./remote_monitoring_demo.sh -t
 
+# To run / restart the demo with a specific UI image
+# RUN:
+$ ./remote_monitoring_demo.sh -u [kruize ui image]
+# RESTART:
+$ ./remote_monitoring_demo.sh -r -u [kruize ui image]
```
Comment (dinogun) [inline remote_monitoring_demo/README.md:41]: This shouldn't be here, it should be part of usage above and ui image should be optional

Comment (bharathappali) [reply remote_monitoring_demo/README.md]: made changes as suggested.  Thanks for the review @dinogun

**Review by dinogun — APPROVED** [review]
LGTM

**Pull request merged**